.scrutinizer.yml

```diff
@@ -2,6 +2,7 @@ filter:
     excluded_paths:
         - 'ieducar/tests/*'
         - 'ieducar/misc/**/*'
+        - '/ieducar/intranet/scripts/jquery'
```

Comment:
Este caminho esta certo? Com a barra no início ele estaria buscando na raiz do OS

Reply:
Segundo o scruntinizer parece que foi aceito, como é uma instancia docker que empurra para a raiz do container